AEPMessaging/Sources/Surface.swift

```diff
@@ -29,6 +29,31 @@ public class Surface: NSObject, Codable {
     }
 
     public init(path: String) {
+        guard !path.isEmpty else {
+            uri = ""
+            return
+        }
```

Comment:
wondering if it's better to throw when `path` is empty in this case.  by definition it can't be a `Surface` without a uri, right?

Reply:
Not sure if constructor throwing would be a good experience with guard/ if everywhere they instantiate the class.
```
guard let surface = try? Surface(path:"") else {...}
```
We do check for valid surfaces before we use them and I think it to be a rare scenario (providing empty paths) to build these extra checks. What do you think?